pipeline/tasks/gapic_tasks.py

```diff
@@ -165,6 +165,11 @@ def execute(self, gapic_api_yaml):
 
 class GapicPackmanTask(packman_tasks.PackmanTaskBase):
     def execute(self, language, api_name, final_repo_dir):
+        # Some APIs will be a part of gcloud project for Ruby and NodeJS.
+        # Such APIs don't need packman.
+        if ((language is 'ruby' or language is 'nodejs') and
```

Comment:
Is `is` brittle here because it's testing pointer equality rather than content equality?